### PR TITLE
UCI Image

### PR DESCRIPTION
**Pull request by mguetta1**
Code changes in order to support UCI VM as conversion host

### CODE REVIEW

File: roles/v2v.ci/files/conv_vm_start_log.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    	echo "Proccess: $proc is not running!"
+	success=false
+    fi
+done
+
+if $success; then
+    echo "+++ Validation passed +++"
+else
+    echo "!!! Validation failed !!!"
+fi
```
Comment (istein1) [inline roles/v2v.ci/files/conv_vm_start_log.sh]: No new line here

File: roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml [hunk 2]
```diff
@@ -66,14 +67,19 @@
         ansible_ssh_pass: "{{ v2v_ci_rhv_root_password }}"
         ansible_pipelining: true
 
-    - name: Read RHV Engine SSH Private key
+    - name: Read RHV Engine SSH RSA Private key file /root/.ssh/id_rsa
```
Comment (istein1) [inline roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml:75]: We will work with this key for now, as the ansible playbook, that created the conversion host VM was run from RHV engine, 
but later we'll need to replace it with key from the flow node, that will run this ansible playbook.

File: roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml [hunk 3]
```diff
@@ -66,14 +67,19 @@
...
       delegate_to: "{{ v2v_ci_rhv_hostname }}"
-      shell: tail -n +4 /etc/pki/ovirt-engine/keys/engine_id_rsa
+      shell: tail -n +4 /root/.ssh/id_rsa
       register: ssh_key
 
-#    - debug: msg="{{ssh_key.stdout}}"
```
Comment (istein1) [inline roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml:74]: Need to remove commented code.

File: roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml [hunk 4]
```diff
@@ -66,14 +67,19 @@
...
+    - name: Read RHV Engine CA certificates bundle file /etc/pki/ovirt-engine/apache-ca.pem
+      delegate_to: "{{ v2v_ci_rhv_hostname }}"
+      shell: tail -n +4 /etc/pki/ovirt-engine/apache-ca.pem
+      register: ca_certs_file
+
+#    - debug: msg="{{ca_certs_file.stdout}}"
```
Comment (istein1) [inline roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml]: Remove commented code.

**Comment by mguetta1** [comment]
Closing PR due some changes that should be added (according to this PR comments)

**Pull request closed without merging**